servicetest/testframework/lib.py

```diff
@@ -109,19 +110,25 @@ def start(self, data):
             {
                 Container.CONFIG: "{enableWriteBuffer: false}",
                 Container.HOST_PATH: my_path_string_variable,
-                Container.BIND_MOUNT_DIR: "app"
+                Container.BIND_MOUNT_DIR: "app",
+                Container.READONLY: True
             }
 
             The values in the dict are passed as arguments to the D-Bus methods on SoftwareContainerAgent.
             The dict is mapped like so:
             Container.CONFIG - argument to SoftwareContainerAgent::CreateContainer
             Container.HOST_PATH - second argument to SoftwareContainerAgent::BindMountFolderInContainer
             Container.BIND_MOUNT_DIR - third argument to SoftwareContainerAgent::BindMountFolderInContainer
+            Container.READONLY - fourth argument to SoftwareContainerAgent::BindMountFolderInContainer
         """
+        # Set some defaults if the user did not pass all values in the data dict
+        if Container.READONLY not in data:
+            data[Container.READONLY] = False
```

Comment:
This is a weird way of setting default values. Either set it in the function or make it mandatory. This will easily be missed by users of the lib.